src/extensions/bigbang/manifests.go

```diff
@@ -67,7 +67,7 @@ func manifestZarfCredentials(version string) corev1.Secret {
 	values := bbV1ZarfCredentialsValues
 
 	semverVersion, err := semver.NewVersion(version)
-	if err != nil && semverVersion.Major() == 2 {
+	if err == nil && semverVersion.Major() == 2 {
 		values = bbV2ZarfCredentialsValues
```

Comment:
I don't think this is necessary.  The naming convention change that is breaking in 2.0 is backward compatible.

Reply:
I'm double checking this.

Reply:
Nevermind, it wasn't for all of them.

Reply:
(I was going off of the latest values.yaml for 1.57.1 which still uses `kyvernopolicies` https://repo1.dso.mil/big-bang/bigbang/-/blob/1.57.1/chart/values.yaml#L412)